### ISSUE TITLE
Improvised Grappler Gloves cannot be crafted.

### ISSUE THREAD

**Issue by LSommers** [citadel-station/citadel-main#13848]
Issue reported from  Round ID: 25425 (Citadel Station)
Reporting client version: 513.1539

I've mentioned this a few times and was told to make a git issue, so here it is. I'm no coder, but I noticed something odd in the code for crafting the DIY grappler gloves in that there's an extraneous comma not present in other crafting recipes after the last required item.

` /obj/item/stack/sheet/cloth = 2,
    )`

I'd triple-checked everything else, hence why I was checking the code itself.

**Comment by DeltaFire15** [comment]
Just tested it on local, seemed to work fine. Did you have wirecutters when attempting to make them?

**Comment by IronEleven** [comment]
My guess is it's that issue where the crafting menu doesn't work until you switch tabs.

**Comment by LSommers** [comment]
> Just tested it on local, seemed to work fine. Did you have wirecutters when attempting to make them? 
         Yes, tried it with both in-hand wirecutters, wirecutters on a table with the items, and wirecutters on the floor with the items.
> My guess is it's that issue where the crafting menu doesn't work until you switch tabs.
         If you mean tabs within the crafting menu itself, that's not it either. It's bizarre. The crafting button simply remains grey.
